config/v1/types_feature.go

```diff
@@ -106,8 +106,10 @@ var FeatureSets = map[FeatureSet]*FeatureGateEnabledDisabled{
 		Disabled: []string{},
 	},
 	TechPreviewNoUpgrade: newDefaultFeatures().
-		with("CSIDriverAzureDisk"). // sig-storage, jsafrane
-		with("CSIDriverVSphere").   // sig-storage, jsafrane
+		with("CSIDriverAzureDisk").    // sig-storage, jsafrane, OCP specific
+		with("CSIDriverVSphere").      // sig-storage, jsafrane, OCP specific
```

Comment:
Why `CSIDriverVSphere` and `CSIDriverAzureDisk`? Don't we want only EBS and Cinder?

Reply:
These are for CSI driver installation, not for migration

Reply:
Can you update the title/description?

Reply:
I am only adding comments to existing feature gates here.

Reply:
Sorry, you just updated the comments